Cargo.lock

```diff
@@ -115,7 +115,7 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "0df2f85c8a2abbe3b7d7e748052fdd9b76a0458fdeb16ad4223f5eca78c7c130"
 dependencies = [
  "addr2line",
- "cfg-if",
+ "cfg-if 0.1.10",
```

Comment:
Are all these changes in `Cargo.lock` due to updating tokio and hyper?

Reply:
Yes, i did `cargo update -p` for tokio and hyper